### PR TITLE
Improve formatting of failed jobs admin page

### PR DESCRIPTION
**Pull request by gylaz**
Make it so things are visible on the screen, rather than scrolling
horizontally.

Uses features from the latest Administrate release, to provide custom
CSS styles.

### CODE REVIEW

File: app/assets/stylesheets/admin/administrate.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.collection-data {
+  th {
+    td {
```
Comment (gylaz) [inline app/assets/stylesheets/admin/administrate.scss]: I think I'm okay with this.

File: config/initializers/administrate.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Rails.application.config.assets.precompile += %w( admin/administrate.css )
```
Comment (thorncp) [inline config/initializers/administrate.rb]: Is this a requirement for the administrate 0.2.0?

**Comment by thorncp** [comment]
:+1: :trumpet:

File: config/initializers/administrate.rb (hunk 2 above)
Comment (gylaz) [inline config/initializers/administrate.rb]: Yes... ?

Without this I get the following error:
screenshot 2016-05-27 10 30 04

**Pull request merged**